src/components/navigation/QuestionTopBar.tsx

```diff
@@ -121,22 +144,46 @@ const QuestionTopBar = ({ additionalSteps }: Props) => {
         </Button>
       </Grid>
       <Grid item>
-        <Stepper
-          data-cy={QUESTION_BAR_CY}
-          nonLinear
-          alternativeLabel
-          activeStep={currentIdx}
-          sx={{ pb: 3 }}
-        >
-          {order?.map((qId: string, index: number) => (
-            <Step key={qId} data-cy={buildQuestionStepCy(qId)} className={
-              QUESTION_STEP_CLASSNAME
-            }>
-              {renderLabel(qId, index)}
-            </Step>
-          ))}
-          {additionalSteps}
-        </Stepper>
+        <DragDropContext onDragEnd={handleDrop}>
+          <Droppable droppableId="list-container" direction="horizontal">
+            {(provided) => (
+              <div
+                className="list-container"
+                {...provided.droppableProps}
+                ref={provided.innerRef}
+              >
+                <Stepper
+                  data-cy={QUESTION_BAR_CY}
+                  nonLinear
+                  alternativeLabel
+                  activeStep={currentIdx}
+                  {...provided.droppableProps}
+                  ref={provided.innerRef}
+                  sx={{ pb: 3 }}
+                >
+                  {order?.map((qId: string, index: number) => (
+                    <Draggable key={qId} draggableId={qId} index={index}>
+                      {(provided) => (
+                        <Step
+                          key={qId}
+                          data-cy={buildQuestionStepCy(qId)}
+                          className={QUESTION_STEP_CLASSNAME}
+                          ref={provided.innerRef}
```

Comment:
It's defined here, in `Stepper` and in `div`?

Reply:
Correct, I removed the additional div as the drag doesn't require it.